build_projects/Microsoft.DotNet.Cli.Build.Framework/CurrentPlatform.cs

```diff
@@ -78,10 +87,15 @@ public static bool IsLinux
         {
             get
             {
-                return IsUbuntu || IsCentOS || IsRHEL || IsDebian;
+                return IsUbuntu || IsCentOS || IsRHEL || IsDebian || IsFedora;
             }
         }
 
+        public static bool IsPlatform(BuildPlatform platform, string version = null)
+        {
+            return IsPlatform(platform) && IsVersion(version);
```

Comment:
I think this needs to be `IsPlatform(platform) && (version == null || IsVersion(version))` otherwise `[BuildPlatforms(BuildPlatform.Ubuntu)]` no longer means all Ubuntus...